Libraries/Animated/src/Interpolation.js

```diff
@@ -163,6 +165,16 @@ function interpolate(
   return result;
 }
 
+function colorToRgba(
+  input: string
+): string {
+  var color = tinycolor(input);
+  var {r, g, b, a} = color.toRgb();
```

Comment:
Can you only declare the variable if it is valid. Otherwise calling toRGB is not safe